apis/python/src/tiledbsoma/sparse_nd_array.py

```diff
@@ -161,9 +162,8 @@ def nnz(self) -> int:
 
     def read(
         self,
-        coords: Optional[SparseNdCoordinates] = None,
-        # *,
-        # TODO: missing parameters
+        slices: Optional[SparseNdCoordinates] = None,
```

Comment:
This is a note to myself and **not** a change that you need to make: we should probably make sure that we have `coords` vs. `slices` standardized within `somacore` (and then we can pick it up from there), and then fix the implementation up once we make that consistent.